model/src/main/java/org/projectnessie/api/http/HttpTreeApi.java

```diff
@@ -408,7 +421,7 @@ void transplantCommitsIntoBranch(
     @APIResponse(responseCode = "404", description = "Ref doesn't exists"),
     @APIResponse(responseCode = "409", description = "update conflict")
   })
-  void mergeRefIntoBranch(
+  MergeResponse mergeRefIntoBranch(
```

Comment:
nit: Could you add OpenApi examples for the new output?

Reply:
I wanted to comment on that as well and forgot, so +1 on adding an example

Reply:
Just realized that there are a bunch of examples that aren't linked anywhere. WIll do it in a follow-up.